src/garrysmod/gamemodes/zombiesurvival/scripting and addons.txt

```diff
@@ -62,9 +62,6 @@ GM:GiveDefaultOrRandomEquipment(Player pl)
 Hook and function. Gives a player random equipment without checking their favorites.
 GM:GiveRandomEquipment(Player pl)
 
-Hook and function. Loads the next map. HERE IS WHERE YOU PUT YOUR MAP VOTING AND STUFF. RETURN TRUE TO OVERRIDE.
-GM:LoadNextMap()
-
```

Comment:
I thought about adding an option to toggle between map cycles and map voting, but since we can't expect everyone to have a PostgreSQL database, I don't think it matters worrying about maintaining this feature to be merged upstream. I'm just gutting the map cycle feature from our fork.